base/common/src/main/java/com/netscape/certsrv/cert/CertReviewResponse.java

```diff
@@ -290,4 +258,246 @@ public boolean equals(Object obj) {
                 && Objects.equals(requestStatus, other.requestStatus) && Objects.equals(requestType, other.requestType);
     }
 
+    public Element toDOM(Document document) {
+
+        Element element = document.createElement("certReviewResponse");
```

Comment:
I did a diff between the results or the unit test on master and your branch and an upper case C was one of the differences 
```suggestion
        Element element = document.createElement("CertReviewResponse");
```